src/main/resources/templates/editMATLABCommandTask.ftl

```diff
@@ -2,3 +2,10 @@
 [@ww.select cssClass="builderSelectWidget" labelKey="executable.type" name="matlabExecutable"
 extraUtility=addExecutableLink  list=uiConfigSupport.getExecutableLabels('matlab') required='true'/]
 [@ww.textfield labelKey='matlab.plugin.command' name='matlabCommand' cssClass="long-field" description="Specify the MATLAB command to execute." required='true'/]
+[@ui.bambooSection]
+    [@ww.checkbox labelKey='matlab.options' name='optionsChecked'  toggle='true'/]
+        [@ui.bambooSection dependsOn='optionsChecked' showOn=true]
+            [@ww.textfield labelKey='matlab.options.field' cssClass="long-field" name="matlabOptions"
```

Comment:
Instead of having it below the command text field, having it below the Executable text field is preferred. See the previous comment for reasoning.